src/test/java/org/swarm/generator/rest/EndpointFilePathGeneratorWithExplicitPackageTest.java

```diff
@@ -0,0 +1,55 @@
+package org.swarm.generator.rest;
```

Comment:
Maybe move to the `io.thorntail.generator` package?

Reply:
Good point, I shall do that :-)